FirebaseDev.podspec

```diff
@@ -1,6 +1,6 @@
 Pod::Spec.new do |s|
   s.name             = 'FirebaseDev'
-  s.version          = '4.0.0'
+  s.version          = '4.0.1'
```

Comment:
This doesn't matter for now. Since we're now planning to have a separate non-interoperable FirebaseDev pod, I'm planning to change this to 0.0.1 later this week.